doc/developer/bestPractices/README

```diff
@@ -9,7 +9,7 @@ developers.  A possible reading order is roughly as follows:
 GettingStarted.txt
   A list of suggested "first steps" for new developers.
 
```

Comment:
Is GitCheatSheet missing an entry here?  If so, would you add one while here?  (or else pull the GitCheatSheet renaming into a separate commit and do it there).